pages/instructorQuestions/instructorQuestions.ejs

```diff
@@ -1,136 +1,228 @@
-                        widthFixed: true,
-                        headerTemplate: '{content} {icon}',
-                        widgets: ["uitheme", "filter", "zebra"],
-                        widgetOptions: {
-                            zebra: ["even", "odd"],
-                            filter_reset: "#resetFilter",
-                            filter_cssFilter: "form-control",
-                            filter_functions: {
-                                3: {
-                                    <% all_tags.forEach(function(tag) { %>
-                                    "<%= escapeRegExp(tag.name) %>": function(e, n, f, i, $r, c, data) {
-                                        return /\b<%= escapeRegExp(tag.name) %>\b/.test(e);
-                                    },
-                                    <% }); %>
-                                },
-                                <% if (locals.course_instance) { %>
-                                5: {
-                                    <% all_assessments.forEach(function(assessment) { %>
-                                    "<%= escapeRegExp(assessment.label) %>": function(e, n, f, i, $r, c, data) {
-                                        return /\b<%= escapeRegExp(assessment.label) %>\b/.test(e);
-                                    },
-                                    <% }); %>
-                                },
-                                <% } %>
-                            },
-                        },
-                    });
-                });
-            </script>
+          </div>
+        </div>
+
+        <table id="questionsTable"
+               data-data="<%= JSON.stringify(questions) %>"
+               data-classes="table table-sm table-hover table-bordered"
+               data-thead-classes="thead-light"
+               data-filter-control="true"
+               data-show-filter-control-switch="true"
+               data-show-search-clear-button="true"
+               data-show-columns="true"
+               data-show-columns-toggle-all="true"
+               data-show-button-text="true"
+               data-pagination="true"
+               data-pagination-v-align="top"
+               data-page-list="[50,100,200,500,unlimited]"
+               data-page-size="50"
+               data-smart-display="false"
+               data-show-extended-pagination="true"
+               data-toolbar=".fixed-table-pagination"
+               data-sticky-header="true">
+          <thead>
+            <tr>
+              <th data-field="qid"
+                  data-sortable="true"
+                  data-class="align-middle sticky-column"
+                  data-formatter="qidFormatter"
+                  data-filter-control="input"
+                  data-switchable="true">QID</th>
+              <th data-field="title"
+                  data-sortable="true"
+                  data-class="align-middle text-nowrap"
+                  data-filter-control="input"
```

Comment:
The title filter seems to be broken for me (the QID filter works fine). For example:

<img width="492" alt="image" src="https://user-images.githubusercontent.com/875833/128927287-83d0c8dd-4156-4201-8b04-e211cb90dc07.png">

<img width="642" alt="image" src="https://user-images.githubusercontent.com/875833/128927317-0beb4c8c-9b05-4e7f-bf06-68a2d4b03273.png">

Reply:
This was a leftover from the custom search used when the title had a potential popover/issues count, which now moved to QID. Fixed.